### DISCUSSION TITLE
[create-next-app] scores poorly with Lighthouse

### DISCUSSION THREAD

**Discussion by SharadKumar** [vercel/next.js#16227] [Help] [answered]
# Feature request

## Is your feature request related to a problem? Please describe.
Lighthouse score for CNA is quite low. It would be good to set a good benchmark, to demonstrate.

## Describe the solution you'd like

Achieve highest possible score, as a starting point for users of create-next-app.

## Additional context

![image](https://user-images.githubusercontent.com/5488144/90333353-77a6c600-e008-11ea-9b4b-b89e16e35290.png)

**Comment by timneutkens** [comment, marked as answer]
You're testing against the development build, not production. run `npm run build` and then `npm run start` to run a correct test.
